GVRf/Sample/gvr-sample/src/org/gearvrf/sample/SampleCubeScript.java

```diff
@@ -160,23 +153,20 @@ public void onInit(GVRContext gvrContext) {
     @Override
     public void onStep() {
         FPSCounter.tick();
-        mFrontBoardObject.getRenderData().getMaterial().setOpacity(1.0f);
-        mFrontBoardObject2.getRenderData().getMaterial().setOpacity(1.0f);
-        mFrontBoardObject3.getRenderData().getMaterial().setOpacity(1.0f);
+        mFrontFace.getRenderData().getMaterial().setOpacity(1.0f);
```

Comment:
This issue as well as the above code-repeat issue are both from the original gvrsample. I agree that there are lots of spaces to optimize it. But it has nothing to do with the 3D screenshot, and thus I would like to do the optimization in a separate commit and submit it later. This will keep each commit clearer and easy for maintain.

In fact, my only change to gvrsample is the screenshot processing methods. All the changes to onInit (as well as corresponding changes in GVRSceneObject) are not related to 3D screenshot, and should be done in a separate commit. But since they are already integrated, I think it is OK to leave them as is. But all further changes should be in a separate commit and should not block this 3D screenshot commit.